4-projects/business_unit_2/shared/variables.tf

```diff
@@ -42,3 +42,9 @@ variable "remote_state_bucket" {
   description = "Backend bucket to load Terraform Remote State Data from previous steps."
   type        = string
 }
+
+variable "enable_cloudbuild_deploy" {
```

Comment:
Rather than exposing a var, can we use one of the [jenkins output ](https://github.com/terraform-google-modules/terraform-example-foundation/blob/253de5e03edf8105ba9262758438e1e6777d494a/0-bootstrap/outputs.tf#L119)from bootstrap to simplify this?

Reply:
Done